wandb/sdk/wandb_settings.py

```diff
@@ -669,6 +721,13 @@ def _validate_anonymous(value: str) -> bool:
             raise UsageError(f"Settings field `anonymous`: '{value}' not in {choices}")
         return True
 
+    @staticmethod
+    def _validate_api_key(value: str) -> bool:
+        if value.startswith(" ") or value.endswith(" "):
```

Comment:
```suggestion
        if len(value) - len(value.strip()) > 0:
```

Reply:
```
"\t my string".startswith(" ")
```
will fail... don't know if we care of other types of whitespaces

Reply:
yea, I changed that to string.whitespace while you were typing your comment, but I like doing it your way even more!

Reply:
haha... great minds 😄